views/agency-requests/edit_participant.php

```diff
@@ -100,26 +102,22 @@ function getPhoneNumbers($phoneTypes){
 
 
  if ($_SERVER['REQUEST_METHOD'] == 'POST') {
-
+     
+    // Count any failed queries
+	$error = 0;
+    
+    // Participant Name Information
 	$firstname = $_POST['fname-update'];
 	$lastname = $_POST['lname-update'];
 	$middleinit = $_POST['mname-update'];
 	
-	//makes sure the middle name is only 1 character
+	// Forces the middle name is only 1 character
 	if(strlen($middleinit) <= 1 ){
 		$res = $db->query("UPDATE people SET firstname = $1, lastname = $2, middleinit =$3 ".
                 "WHERE peopleid = $4", [$firstname, $lastname, $middleinit, $peopleid]);
 			if(!$res){
-			?>
-				<div class="alert alert-success alert-dismissible fade show" role="alert">
-					<button type="button" class="close" data-dismiss="alert" aria-label="Close">
-						<span aria-hidden="true">&times;</span>
-					</button>
-					<strong>Participant Update Success</strong> Participant name information has been updated.
-				</div>
-
-			<?php
-			}
+                $error++;
+            }
 		
 	}else{
 		?>
```

Comment:
The alert box below counts as output and would cause header redirect issue.

Reply:
Fixed the redirect issue, was a problem with when I was including header.php